src/components/layers/LayerList.jsx

```diff
@@ -184,7 +186,7 @@ class LayerListContainer extends React.Component {
       if(layers.length > 1) {
         const grp = <LayerListGroup
           data-wd-key={[groupPrefix, idx].join('-')}
-          key={`group-${groupPrefix}`}
+          key={`group-${groupPrefix}-${idx}`}
```

Comment:
Groups of layers can appear multiple times, for example

```
water_1
water_2
road
water_3
water_4
```

Would have 2 `water` groups

```
water (group)
road (layer)
water (group)
```